lib/shopify_transporter/exporters/magento/customer_exporter.rb

```diff
@@ -15,6 +15,7 @@ def key
 
         def export
           base_customers.each do |customer|
+            # rubocop:disable RedundantBegin
```

Comment:
I think we need to add an enable after the block/lines of code we care about as well, otherwise cop will skip the rest of the file.

How do you feel about just removing the `RedundantBegin` check from our rules instead? I prefer this approach.

Reply:
I went ahead and did the fix. On second thought, I don't think it's terribly concerning to not have rubocop yell at us if we have redundant begin block.